### PR TITLE
Retire the Canvas repo

### PR DESCRIPTION
**Pull request by athei**
Add a warning that the canvas node has moved. After this is merged we should archive this repo. CI failures should be ignored as we are not building from this repo anymore.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
 
-[a1]: https://gitlab.parity.io/parity/canvas/badges/master/pipeline.svg
-[a2]: https://gitlab.parity.io/parity/canvas/pipelines?ref=master
 [k1]: https://img.shields.io/badge/matrix-chat-brightgreen.svg?style=flat
 [k2]: https://riot.im/app/#/room/#ink:matrix.parity.io
 [l1]: https://img.shields.io/discord/722223075629727774?style=flat-square&label=discord
 [l2]: https://discord.com/invite/wGUDt2p
 
+> :warning: **Note: Canvas was moved into the [Cumulus repo](https://github.com/paritytech/cumulus).
+> Please check there for the current version**
```
Comment (cmichi) [inline README.md:11]:
```suggestion
> :warning: **Note: Canvas was moved into the [Cumulus repo](https://github.com/paritytech/cumulus#canvas-).
> Please check there for the current version.**
```

**Review by cmichi — APPROVED** [review]

**Comment by cmichi** [comment]
We should also make a decision on how to handle the open issues which are still in here.

**Comment by cmichi** [comment]
Also we have to review `ink-docs` and the Substrate contracts workshop for links to this repo.

**Comment by athei** [comment]
> We should also make a decision on how to handle the open issues which are still in here.

Created a new `Canvas` project in the cumulus repo and transferred them there.

> Also we have to review ink-docs and the Substrate contracts workshop for links to this repo.

Yes but no high prio as we will have a link here sending people to the correct repo.

**Review by ascjones — APPROVED** [review]

**Pull request merged**